brave/brave-core
# Remove Brave Ads page classification scores from locally persisted diagnostic log

Resolves https://github.com/brave/brave-browser/issues/11220

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm `Event log` for page classification scores is not logged in the diagnostic log
- Confirm when viewing an ad the event is logged in the diagnostic log
- Confirm when clicking an ad the event is logged in the diagnostic log
- Confirm when dismissing an ad the event is logged in the diagnostic log
- Confirm when an ad times-out the event is logged in the diagnostic log

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
